web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -61,11 +61,12 @@ a, a:hover {
   display: flex;
   flex-direction: column;
   gap: 1px;
-  padding-left: 8px;
+  margin-left: 13px;
+  border-left: 1px solid var(--color-secondary);
 }
 
 .sub-items .item-file {
-  padding-left: 24px;
+  padding-left: 18px;
```

Comment:
And this one too.

Reply:
Perhaps we should document this math, otherwise it will break again and again…

Reply:
certainly

Reply:
I didn't do any particular math when changing these the last time, I just went with what looked okay.